app/jobs/set_license_and_rights_statements_job.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+##
+# Job to set licenses, copyright statements, and/or use & reproduction statements
+class SetLicenseAndRightsStatementsJob < GenericJob
+  # @param [Integer] bulk_action_id GlobalID for a BulkAction object
+  # @param [Hash] params additional parameters that an Argo job may need
+  # @option params [Array] :pids required list of identifiers
+  # @option params [String] :copyright_statement the new copyright statement
+  # @option params [String] :copyright_statement_option option to update the copyright statement
+  # @option params [String] :license the new license value
+  # @option params [String] :license_option option to update the license value
```

Comment:
I don't see this used

Reply:
It's used in a meta-programming-y way on line 44.